src/battle-log.ts

```diff
@@ -210,6 +210,17 @@ export class BattleLog {
 
 		case 'error': case 'inactive': case 'inactiveoff':
 			divClass = 'chat message-error';
+			battle = this.scene?.battle;
+
+			if (battle?.kickingInactive && battle.timerMessageHighlighted) {
+				divClass += ' highlighted';
+				if (this.scene?.battle?.roomid) {
+					const title = 'Make your move!';
+					const body = `${battle?.kickingInactive || 'Only a few'} seconds remaining!`;
```

Comment:
You check `battle?.kickingInactive` in the if statement, so it should always be truthy. Is the "Only a few" fallback just to satisfy TypeScript, or is there a situation where it would be used that I'm not seeing?